app/Http/Controllers/Auth/LoginController.php

```diff
@@ -28,6 +30,23 @@ class LoginController extends Controller
      */
     protected $redirectTo = RouteServiceProvider::HOME;
 
+    /**
+     * Handle an authentication attempt.
+     *
+     * @return Response
+     */
+    public function authenticated()
+    { 
+        $user = Auth::user(); 
+        // user is admin.
+        if ($user->role == 1) {
```

Comment:
không dùng '==' toàn bộ dùng '===' . không dùng trực tiếp value '1' , thay vào đó hãy tạo constant và sử dụng constant name